pages/api/comments.js

```diff
@@ -0,0 +1,25 @@
+import Cors from 'micro-cors'
+import request from 'request'
+
+const cors = Cors({
+  allowedMethods: ['GET', 'HEAD']
+})
+
+const getCommentCount = (req, res) => {
+  const {
+    query: { url }
+  } = req
+
+  request(`${url}.json`, (error, response, body) => {
```

Comment:
What `url` would this typically be? Thinking about possible security vulnerabilities since this is input passed on to `request` without validation.

Reply:
Also (for @shcheklein) do we rate limit API requests somehow?

Reply:
It will be a duscourse forum urls like: https://discuss.dvc.org/t/how-do-i-remove-files-which-are-no-longer-tracked-by-dvc-from-dvc-remote-ssh-storage/113

Basically if you add `.json` to it, it will return you json representation of the topic.

Reply:
@iAdramelk question is still valid - the current setup means that anyone can pass any url and use us a proxy, right?

Reply:
@shcheklein Well. This route returns json in the form of `{ count: NUMBER }` or `{ error: STRING }`. If someone wants to get this json from us by passing their URL to this route, then yes, they can do it.

Reply:
it means for example that someone can hit some malware URL using our server and get us blocked for example?

Reply:
Maybe just validate agains a simple regex `/^https:\/\/discuss.dvc.org\/[a-z\/-0-9]$/` and return HTTP 403 if failed.

Reply:
Added CORS origin and URL validation to prevent it.

Reply:
Great, thanks.